File: java/archbots/nullmage/SelectCurse.java
package archbots.nullmage;

import com.runemate.game.api.osrs.local.hud.interfaces.Magic;
import com.runemate.game.api.script.framework.tree.LeafTask;

public class SelectCurse extends LeafTask {
    @Override
    public void execute() {
        Magic.STUN.activate();
        NullMage.lastSpell = false;
    }
}

File: kotlin/archbots/archmage/treebot/leaves/SelectCurse.kt
package archbots.archmage.treebot.leaves

import archbots.nullmage.NullMage
import com.runemate.game.api.osrs.local.hud.interfaces.Magic
import archapi.treebot.framework.LeafTask

class SelectCurse : LeafTask() {
    override fun execute() {
        Magic.STUN.activate()
        NullMage.lastSpell = false
    }
}
